src/test/java/org/takes/facets/auth/codecs/CcBase64Test.java

```diff
@@ -134,9 +134,24 @@ public void decodesNonBaseSixtyFourAlphabetSymbols() throws IOException {
      * @throws Exception If some problem inside
      */
     @Test
-    public void equalsAndHashCodeEqualTest() throws Exception {
-        EqualsVerifier.forClass(CcBase64.class)
-            .suppress(Warning.TRANSIENT_FIELDS)
-            .verify();
+    public void equalsTest() throws Exception {
+        new Assertion<>(
+            "CcBase64 objects must be equal",
```

Comment:
@fanifieiev we can't rewrite this phrase to begin with "Must..."?

Reply:
@fabriciofx we can